plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/font-pairing-variations/constants.ts

```diff
@@ -225,6 +225,16 @@ export const FONT_PAIRINGS_WHEN_AI_IS_OFFLINE = [
 					},
 				},
 			},
+			blocks: {
+				'core/site-title': {
+					typography: {
+						fontFamily:
+							'var(--wp--preset--font-family--albert-sans)',
+						fontSize: 'var(--wp--preset--font-size--medium)',
+						fontStyle: 'normal',
```

Comment:
Is there a particular reason why are we adding fontStyle on this font pair? If I remember well, `normal` fontStyle should be the default one.

Reply:
Good catch, removed!